django_project/changes/views/category.py

```diff
@@ -506,11 +510,16 @@ def get_queryset(self):
         projects which user created (staff gets all projects)
         :rtype: QuerySet
         """
+        self.project_slug = self.kwargs.get('project_slug', None)
```

Comment:
It's just my advice, but I try to not store all variables in `self`. If I don't store the variable in `self`, at least the next developer coming to read this code, he is aware that it's a local variable to this function and it will not be altered or read in another function. I try to avoid `self` whenever I can.

Reply:
I see, ok I'll fix it